code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -89,7 +89,7 @@
 
 /obj/item/gun/projectile/pistol/throwback
 	name = "pistol"
-	desc = "A product of one of thousands of illegal workshops from around the galaxy. Often replicas of ancient Earth handguns, these guns are usually found in hands of frontier colonists and pirates."
+	desc = "A product of one of the world's hundreds of illicit workshops. These firearms, which are frequently reproductions of past handgun blueprints, are typically seen in the hands of smugglers and low-lifes."
```

Comment:
```suggestion
	desc = "A product of one of the worlds' hundreds of illicit workshops. These firearms, which are frequently reproductions of past handgun blueprints, are typically seen in the hands of smugglers and low-lifes."
```

Reply:
No don't do that, `worlds'` is the plural possessive which implies there are several worlds.